core/liblwm2m.h

```diff
@@ -497,13 +498,22 @@ typedef struct _lwm2m_observed_
     lwm2m_watcher_t * watcherList;
 } lwm2m_observed_t;
 
+typedef enum {
+    NOT_BOOTSTRAPPED = 0,
```

Comment:
equivalent to an empty lwm2m_context_t::serverList (just thinking out loud here)